Add vector cubes (#59)
Open-EO/openeo.org: documentation/1.0/glossary.md

## Spatial datacubes

A spatiotemporal datacube is a multidimensional array with one or more spatial or temporal dimensions.
In the EO domain, it is common to be implicit about the temporal dimension and just refer to them as spatial datacubes in short.
Special cases are raster and [vector datacubes](https://r-spatial.org/r/2022/09/12/vdc.html).
Learn more about datacubes in the [datacube documentation](https://openeo.org/documentation/1.0/datacubes.html).

## Vector data

In general, **vector data** represent specific things (also called "features") in a space, e.g. on the surface of the Earth.

A **coordinate** represents a specific point in space.

A **feature** is usually a geometry (e.g. the outline of an agricultural field, a forest or an urban area) and it may have additional properties assigned (e.g. a name, a color or a population).

**Geometries** consist of one or more coordinates that may be connected and then form a specific type of geometry, e.g. two points can be connected to a straight line and four straight lines can be connected to rectangle.

Commonly used types of geometries are:
- Points
- Lines / LineStrings

> - Polygons (e.g., a Triangle or Rectangle)

Reviewer: Change into:
```
- Point
- LineString (connected straight line pieces)
- Polygon (connected straight line pieces forming a closed ring, possibly with holes)
```

> Multiple geometries of the same type can be combined to a group of geometries, e.g. a Multi Point or a Multi Polygon.

Reviewer 2: ```suggestion
Multiple geometries of the same type can be combined into a group of geometries, e.g. a Multi Point or a Multi Polygon.
```

Author: ```suggestion
Multiple geometries of the same type can be combined into a group of geometries, e.g. a so-called "multi point" or  "multi polygon".
```

> A spatiotemporal datacube is a multidimensional array with one or more spatial or temporal dimensions. In the EO domain, it is common to be implicit about the temporal dimension and just refer to them as spatial datacubes in short. Special cases are raster and vector datacubes. Learn more about datacubes in the [datacube documentation](https://openeo.org/documentation/1.0/datacubes.html).

Author: ```suggestion
A spatiotemporal datacube is a multidimensional array with one or more spatial or temporal dimensions.
In the EO domain, it is common to be implicit about the temporal dimension and just refer to them as spatial datacubes in short.
Special cases are raster and [vector datacubes](https://r-spatial.org/r/2022/09/12/vdc.html).
Learn more about datacubes in the [datacube documentation](https://openeo.org/documentation/1.0/datacubes.html).
```

## Vector data

In general, **vector data** represent specific things (also called "features") in a space, e.g. on the surface of the Earth.

A **coordinate** represents a specific point in space.

> A **feature** is usually a geometry and it may have additional properties assigned, e.g. a name, a color or a temperature (at a certain time).

Reviewer 3: I know this is not a introduction to GIS terminology, but the explanation is pretty vague up to this point (feature = specific things = usually a geometry).

I would add some feature examples, e.g.

> A **feature** is usually a geometry (e.g. the outline of an agricultural field, a forest, an urban area, ...)

Reviewer 3: > a temperature (at a certain time).

This might confuse the reader, because it is unclear if some additional time dimension is involved, or if it's just a one-off temperature measurement. I would drop the `(at a certain time)` and use a  numerical property example that fluctuates less in time, e.g. elevation, population, ...

Author: ```suggestion
A **feature** is usually a geometry (e.g. the outline of an agricultural field, a forest or an urban area) and it may have additional properties assigned (e.g. a name, a color or a population).
```

**Geometries** consist of one or more coordinates that may be connected and then form a specific type of geometry, e.g. two points can be connected to a straight line and four straight lines can be connected to rectangle.

Commonly used types of geometries are:
- Points
- Lines / LineStrings

> - Polygons (e.g. a Traingle or Rectangle)

Author: ```suggestion
- Polygons (e.g. a Triangle or Rectangle)
```

Multiple geometries of the same type can be combined to a group of geometries, e.g. to a so called "Multi Point" or a "Multi Polygon".

> Features and geometries are specified by the OGC in the [Simple Feature Access specification](https://www.ogc.org/standards/sfa) (and ISO 19125). See the specification for more details.

Author: ToDo: Add link to @edzer's r-spatial blog post about vector cubes.
